UnityProject/Assets/Scripts/Items/Bureaucracy/Photocopier.cs

```diff
@@ -26,13 +24,35 @@ public class Photocopier : NetworkBehaviour, ICheckedInteractable<HandApply>, IS
 		private RegisterObject registerObject;
 
 		[SerializeField] private AddressableAudioSource Copier = null;
+		[SerializeField] private ItemStorage inkStorage;
+		[SerializeField] private ItemTrait tonerTrait;
+		[SerializeField] private GameObject tonerPrefab;
+		private Toner inkCartadge;
+		public Toner InkCartadge => inkCartadge;
+
 
 		private void Awake()
 		{
 			photocopierState = PhotocopierState.Idle;
 			registerObject = gameObject.GetComponent<RegisterObject>();
 			printer = new Internal.Printer(0, trayCapacity, false);
 			scanner = new Internal.Scanner(false, true, null, null);
+			if (inkStorage == null) inkStorage = GetComponent<ItemStorage>();
+		}
+
+		private IEnumerator Start()
+		{
+			//We can't spawn stuff into slots on Awake() because it will screech at us
+			//We do this on Start() and after a frame when the game loads.
+			yield return WaitFor.EndOfFrame;
+			if (tonerPrefab == null) yield break;
+			foreach (var slot in inkStorage.GetItemSlots())
+			{
+				if(slot.IsOccupied) continue;
+				Inventory.ServerSpawnPrefab(tonerPrefab, slot);
```

Comment:
Can't you do this with inventory populaters or specify directly in the prefab what to spawn in the item storage?